models/Product.php

```diff
@@ -54,7 +50,7 @@ public function rules()
             [['created_by'], 'exist', 'skipOnError' => true, 'targetClass' => User::class, 'targetAttribute' => ['created_by' => 'id']],
             [['name', 'description'], 'string', 'max' => 255],
             [['content'], 'string'],
-//            [['pictureFile'], 'file', 'skipOnEmpty' => false, 'extensions' => 'png, jpg'],
+            [['pictureFile'], 'safe'],
```

Comment:
why didn't keep the commented one?

Reply:
It has been removed in posterior commits

Reply:
Because i'm not using direct upload anymore, instead i've used Humhub's UploadWidget